plugins/woocommerce-admin/client/products/sections/pricing-section.scss

```diff
@@ -4,3 +4,10 @@
 		margin-top: $gap-small;
 	}
 }
+
+.woocommerce-field-width {
```

Comment:
This class name feels too generic (I recognize we have other generic class names in this file... we should clean those up too in a future PR). Since it is in the pricing section, we should name it something like:

`product-pricing-section__price-field`

Or, move it somewhere more general, like https://github.com/woocommerce/woocommerce/blob/736593ef1526ac1aa5495851789690b26eb2d657/plugins/woocommerce-admin/client/products/product-page.scss and call it something like:

```
.woocommerce-add-product,
.woocommerce-edit-product {
    .half-width-field {
        // props go here
    }
}
```

I'm not sold on the `half-width-field`... part of me likes it and part of me thinks we should list our the classname of every specific type of field that is going to use this instead. What do you think?